ldk/javascript/src/types/oliveHelps/clipboard.d.ts

```diff
@@ -4,8 +4,6 @@ declare namespace Clipboard {
 
     write: Common.ReadableWithParam<string, void>;
 
-    listen: Common.Listenable<string>;
-
-    includeOliveHelpsEvents(enabled: boolean): void;
+    listenAll: Common.Listenable<string>;
```

Comment:
should this be listen?

Reply:
I think the new function we had in `core` is called `listenAll`, so it is correct.

Reply:
my mistake, didnt look at the folder the file was in!